ios-trivia-game/ViewControllers/AnswerViewController.swift

```diff
@@ -9,27 +9,117 @@
 import UIKit
 
 class AnswerViewController: UIViewController {
+    
+    @IBOutlet weak var timerButton: UIBarButtonItem!
+    @IBOutlet weak var questionLabel: UILabel!
+    @IBOutlet weak var answerTableView: UITableView!
+    
+    var question: TriviaQuestion?
+    var roomId: String?
+    
+    fileprivate var answers: [Answer]? = []
+    fileprivate var timerCount = 60
+    fileprivate var countdownTimer = Timer()
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        
+        questionLabel.text = question?.question
+        
+        answerTableView.delegate = self
+        answerTableView.dataSource = self
+        answerTableView.estimatedRowHeight = 100
+        answerTableView.rowHeight = UITableViewAutomaticDimension
+        
+        countdownTimer = Timer.scheduledTimer(timeInterval: 1, target: self, selector: #selector(updateCounter), userInfo: nil, repeats: true)
+        timerButton.title = ""
+        
+        FirebaseClient.instance.getAnswersBy(roomId: roomId!, questionId: (question?.id)!, complete: { (answers) in
+            for answerDict in answers {
+                let answer = Answer(dictionary: answerDict as! NSDictionary)
+                self.answers?.append(answer)
+                
+                Logger.instance.log(logLevel: .info, message: "Adding \(answer.getJson())")
+            }
+            
+            self.answerTableView.reloadData()
+            
+        }, onError: { (error) in
+            Logger.instance.log(logLevel: .error, message: "Could not find answers for roomId=\(self.roomId!), questionId=\((self.question?.id)!)")
+        })
     }
 
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
     
+    @IBAction func onQuitFromAnswer(_ sender: UIBarButtonItem) {
+        // update user_in_game to remove player from user_in_game
```

Comment:
Before we actually quit the game, should we ask the user for confirmation? Just in case they pressed it by mistake or something. We can throw a UIAlertAction here or something. UI doesn't matter as much for now.